shared/constants/teams.js

```diff
@@ -319,6 +320,10 @@ const getTeamMemberCount = (state: TypedState, teamname: Types.Teamname): number
 const isLastOwner = (state: TypedState, teamname: Types.Teamname): boolean =>
   isOwner(getRole(state, teamname)) && !isMultiOwnerTeam(state, teamname)
 
+const canDeleteTeam = (state: TypedState, teamname: Types.Teamname): boolean =>
+  // FIXME: Use getCanPerform().deleteTeam once it exists: CORE-10908.
+  isOwner(getRole(state, teamname)) && !teamHasSubteams(state, teamname)
```

Comment:
We should be able to delete subteams too - I'm not sure what the rules are for that (do you need to be an owner of the root team?)